### DISCUSSION TITLE
what can we say about the model or the dataset , if i have the results of CTC and RNNT giving almost the same WER ?

### DISCUSSION THREAD

**Discussion by 12DEVESH** [NVIDIA-NeMo/NeMo#9617] [General]
so i tried two models 
Conformer-CTC and Conformer-RNNT on the same dataset. though there are many documents on NVIDIA docs which says that RNNT model is better than CTC. but through all my experiments I am getting almost same results for them .

Also training RNNT is costing me latency. 
RNNT takes more time in inference than CTC though RNNT is made for streaming tasks.

attaching some observations :
Dataset-1:
val_wer graphs : (purple is RNNT )
![image](https://github.com/NVIDIA/NeMo/assets/96682968/25d53717-6faf-452f-88b8-ef2cafbefa6d)

inference time in s : (Purple is RNNT)
![image](https://github.com/NVIDIA/NeMo/assets/96682968/bd38391e-d388-4c34-bbba-ed9003766b9f)

Also one thing I noticed is the inference time gap varies with different dataset:
Dataset-2:
Val_wer( purple is RNNT)
![image](https://github.com/NVIDIA/NeMo/assets/96682968/a20c8254-caa6-43bd-ba9e-2aa4bbfec292)

inference time:
![image](https://github.com/NVIDIA/NeMo/assets/96682968/80aac2af-a3ef-4ed8-b6e1-c77fd470ff7f)

both the datasets contains multilingual audios and corresponding transcripts in Romanic form.

**Comment by titu1994** [comment]
This is interesting, normally we find RNNT to be slower but more accurate. What languages are being trained on? Maybe if they are Chinese, Japanese Korean, the vocab size causes slower learning ?
